### PR TITLE
Upgrade ruby to 2.7.2

### PR DESCRIPTION
**Pull request by bencgreenberg**
Ruby 2.5.8 will be approaching end of life during December, so it's time to upgrade. Ruby 2.7.2 incorporates the Ruby 2.7 updates and addresses security vulnerabilities present in 2.7 and 2.7.1.

### CODE REVIEW

File: lib/nexmo_developer/app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -197,7 +197,7 @@ def coverage_from_files
 
       parts.insert(1, 'top-level') if parts.count < 3
 
-      parts = [parts[0], parts[1], parts[2..-1].join('/')] if parts.count > 3
+      parts = [parts[0], parts[1], parts[2..].join('/')] if parts.count > 3
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/controllers/dashboard_controller.rb:200]: why is this necessary?

Comment (bencgreenberg) [reply lib/nexmo_developer/app/controllers/dashboard_controller.rb:200]: its a rubocop cop, we can disable the cop if you'd prefer?

**Review by fabianrbz — APPROVED** [review]
